src/keywords.cpp

```diff
@@ -309,16 +309,16 @@ void init_keywords()
 
 static int kw_compare(const void *p1, const void *p2)
 {
-   const chunk_tag_t *t1 = (const chunk_tag_t *)p1;
-   const chunk_tag_t *t2 = (const chunk_tag_t *)p2;
+   const chunk_tag_t *t1 = reinterpret_cast<const chunk_tag_t *>(p1);
```

Comment:
Shouldn't static_cast<> work here?  p1 is a (void *).
This comment applies to all conversions to/from (void *).

http://www.cplusplus.com/doc/tutorial/typecasting/